controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -2079,6 +2079,11 @@ func (r *BareMetalHostReconciler) updateEventHandler(e event.UpdateEvent) bool {
 		return true
 	}
 
+	// The assumption that this will not always be filled out is why this check is not appended to the below block
+	if oldBMH.Status.LastUpdated != nil && newBMH.Status.LastUpdated != nil && oldBMH.Status.LastUpdated != newBMH.Status.LastUpdated && reflect.DeepEqual(oldBMH.Spec, newBMH.Spec) && reflect.DeepEqual(oldBMH.ObjectMeta, newBMH.ObjectMeta) {
```

Comment:
Why only skip reconcile when LastUpdated has changed?

Naively I'd have thought we'd want to skip it _even more_ when LastUpdated has **not** changed.

Reply:
> Running the test that I added as part of this change works without the patch included. So I can dismiss this change and leave the test but we probably should close the issue if it is fixed.

This removes the included patch but leaves the test.  Resolving